gitleaves/reports.py

```diff
@@ -68,16 +72,29 @@ def get_next_leaves_by_month(bydates):
     return nextbymonths
 
 
-def export_month_csv(year, month):
-    leaves_csv_path = leaves_csv_path_pat.format(YYYY=year)
-    data = load_csv(leaves_csv_path)
-    outfile = f'/tmp/leaves.{year}.{month}.csv'
-    outdata = ((d.isoformat(), *names)
-               for d, names in
-               data['bymonths'][month].items())
-    csv.writer(open(outfile, 'w')).writerows(outdata)
-    print(f'Data exported to {outfile}')
+# Shekhar’s original in case Jason messes up, — mjb 12/20/2021
+# def export_month_csv(year, month):
+#     leaves_csv_path = leaves_csv_path_pat.format(YYYY=year)
+#     data = load_csv(leaves_csv_path)
+#     outfile = f'/tmp/leaves.{year}.{month}.csv'
+#     outdata = ((d.isoformat(), *names)
+#                for d, names in
+#                data['bymonths'][month].items())
+#     csv.writer(open(outfile, 'w')).writerows(outdata)
+#     print(f'Data exported to {outfile}')
+
+
+def export_month_csv(some_group_of_people):
+    people_by_month = some_group_of_people['group_of_people_by_month']
```

Comment:
Naming suggestions
- some_group_of_people -> data
- group_of_people_by_month -> bymonthnames_groupedby_applicants / bymonthnames_byapplicants

Reply:
lessons learnt
1. Make function parameters as generic/abstract as possible
2. Make variables as descriptively precise as you can possibly make them

Changes done.